src/plugins/LADTable/LADTableSummaryViewProvider.js

```diff
@@ -0,0 +1,68 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define([
+    './components/LADTableSummary.vue',
+    'vue'
+], function (
+    LADTableSummaryComponent,
+    Vue
+) {
+    function Tabs(openmct) {
+        return {
+            key: 'LADTableSummary',
+            name: 'LAD Table Summary',
+            cssClass: 'icon-tabular-lad',
+            canView: function (domainObject) {
+                return domainObject.type === 'LADTableSummary';
+            },
+            view: function (domainObject) {
+                let component;
+
+                return {
+                    show: function (element) {
+                        component =  new Vue({
+                            components: {
+                                LADTableSummaryComponent: LADTableSummaryComponent.default
+                            },
+                            provide: {
+                                openmct,
+                                domainObject,
+                                composition: openmct.composition.get(domainObject)
```

Comment:
We use injectors in the plugin for injecting openmct and domainObject, and I expect we will find a better way to do this in the future.  In this case, you're also passing composition via injection, and it would be more consistent with the rest of the app if you set composition in `data` method or `mounted` function of the component.